server/pulp/server/managers/repo/publish.py

```diff
@@ -340,8 +340,10 @@ def publish_history(self, repo_id, distributor_id, limit=None, sort=constants.SO
 
         # Retrieve the entries
         cursor = RepoPublishResult.get_collection().find(search_params)
-        # Sort the results on the 'started' field. By default, descending order is used
-        cursor.sort('started', direction=constants.SORT_DIRECTION[sort])
+        # Sort the results on the ObjectId field. This will effectively sort by when the item was
+        # created. This is preferred because 'started' is not indexed. By default, descending order
```

Comment:
"This is preferred because 'started' is not indexed" I don't think this comment will be effective for person who is looking at this code for the first time unless he will look at the history of this file and find out that previously it was sorted by 'started' field.
I think more efficient would be to write something like this instead: This way of sorting is preferred to non indexed fields, as those can lead to the database error: too much data for sort() with no index.